css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  width: 100%;
+}
```

Comment:
I would like to see you use the positioning properties for this as well rather than `width` (top, right, bottom, and/or left)